src/app_state.rs

```diff
@@ -99,11 +102,11 @@ pub async fn init_agent() -> Agent {
     #[cfg(feature = "local-bin")]
     {
         let agent = Agent::builder()
-            .with_url("http://127.0.0.1:4943")
+            .with_url("https://ic0.app")
             .build()
             .unwrap();
 
-        agent.fetch_root_key().await.unwrap();
+        // agent.fetch_root_key().await.unwrap();
 
```

Comment:
Any reason we are changing this? wouldn't it break test containers?

Reply:
currently off-chain agent is not used in test-containers

Reply:
else we need to run dfx everytime